frontend/src/components/commons/TabNavigation.jsx

```diff
@@ -46,14 +62,28 @@ class TabNavigation extends Component {
     this.setState({ currentTab: id, currentBody: this.props.tabSpecs[id].body });
   }
 
+  // this function is simply calling 'isTabIdDisabled' function with the needed parameters
+  handleDisabledTabCheck = tabId => {
```

Comment:
You could just say:
```
  handleDisabledTabCheck = tabId => {
   return isTabIdDisabled(this.props.disabledTabsArray, tabId);
  };
```

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/251/commits/7201cc86b1d941ebcb235bb2df22119aa903a2b9